### ISSUE TITLE
Channel Names should update with label

### ISSUE THREAD

**Issue by mo-gaafar** [mo-gaafar/Multi_Signal_Visualizer#19]

**Comment by mo-gaafar** [comment]
fixed in refactoring2

**Comment by mo-gaafar** [comment]
need update the combobox too

**Comment by Devyash29** [comment]
Hello , I've uploaded the hex file from dataset but it is not working. I can't see any signal. Please help me out

**Comment by mo-gaafar** [comment]
> Hello , I've uploaded the hex file from dataset but it is not working. I can't see any signal. Please help me out

Have you tried the files ending with .dat, as shown in the demo? as some files unfortunately may cause errors depending on the format

**Comment by Devyash29** [comment]
Yes I've tried both the files provided in the dataset but no able to see
any waveforms.

**Comment by mo-gaafar** [comment]
Try pulling the latest commit now, it is fixed

**Comment by Devyash29** [comment]
Sure thank you it's working now!

**Comment by mo-gaafar** [comment]
You are welcome, if you found this project of value feel free to modify and add pull requests
________________________________
